pkg/ingester/client/client.go

```diff
@@ -20,6 +22,19 @@ var ingesterClientRequestDuration = promauto.NewHistogramVec(prometheus.Histogra
 	Help:      "Time spent doing Ingester requests.",
 	Buckets:   prometheus.ExponentialBuckets(0.001, 4, 6),
 }, []string{"operation", "status_code"})
+var inflightRequestCount = promauto.NewGaugeVec(prometheus.GaugeOpts{
+	Namespace: "cortex",
+	Name:      "ingester_client_request_count",
```

Comment:
```suggestion
	Name:      "ingester_client_inflight_push_requests",
```
WDYT?. Definitely not ending in _count

Reply:
Sounds good to me. I will make the change.